libs/cli/src/generators/base/generator.ts

```diff
@@ -49,6 +48,7 @@ async function generateEntrypointFiles(tree: Tree, alias: string, options: HlmBa
 			json.compilerOptions.paths[alias] = [`./${joinPathFragments(targetLibDir, 'index.ts').replace(/\\/g, '/')}`];
 			return json;
 		});
+		generateFiles(tree, path.join(__dirname, '..', 'ui', 'libs', options.internalName, 'files'), targetLibDir, options);
```

Comment:
Error is gone, but buildable entry points don't have the ui component files copied over. I moved this below the if statement and now the files are copied for (non)-buildable